packages/datacommons-mcp/datacommons_mcp/topics.py

```diff
@@ -114,6 +119,27 @@ def get_name(self, dcid: str) -> str:
         """Get the human-readable name for a DCID."""
         return self.dcid_to_name.get(dcid, "")
 
+    def merge(self, other: Self) -> Self:
+        """Merge another TopicStore into this one.
+
+        For overlapping data, this store's data prevails.
+        Only new data from the second store is added.
+        """
+        # Only add topics that don't already exist
+        for topic_dcid, topic_data in other.topics_by_dcid.items():
+            if topic_dcid not in self.topics_by_dcid:
```

Comment:
Is this so that custom topics can essentially "override" base topics? As opposed to merging the variables within the topics.